README.md

```diff
@@ -151,3 +151,7 @@ Below you can find information on how to push local code changes to a kind clust
 - push image to local registry: `podman push localhost:5000/maiqueb/multus-dynamic-networks-controller`
 - update manifests to use the generated image: `IMAGE_REGISTRY=localhost:5000/maiqueb make manifests`
 - deploy the controller: `kubectl apply -f manifests/dynamic-networks-controller.yaml`
+
+## Known limitations
```

Comment:
So we allow hot-unplug of cold-plugged (?) interfaces?

Reply:
Currently ? Yes. 

Do we want to ?... Not sure. 

I'm inclined to say no.

Reply:
Ack